### PR TITLE
trailscraper 0.7.0

### PR DESCRIPTION
**Pull request by flosell**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

just released, new version of [trailscraper](https://github.com/flosell/trailscraper), resources generated using ` homebrew-pypi-poet`

### CODE REVIEW

File: Formula/trailscraper.rb [hunk 1]
```diff
@@ -3,11 +3,9 @@ class Trailscraper < Formula
 
   desc "Tool to get valuable information out of AWS CloudTrail"
   homepage "https://github.com/flosell/trailscraper"
-  url "https://files.pythonhosted.org/packages/ee/5f/67657d646b3a6fdd9059f72cf9f13cb9ed677d92c881bc83b67179c20178/trailscraper-0.6.4.tar.gz"
-  sha256 "3e076fc97addddfd8b64ea51a6cc9e2af9d9bffcc68423772ff6cb7b63a009d6"
+  url "https://github.com/flosell/trailscraper/archive/0.7.0.tar.gz"
+  sha256 "39fc2895629242dc72e0f3d3665b7fe53cc1596228578f5f7384cc6ea09d353e"
   license "Apache-2.0"
-  revision 1
-  head "https://github.com/flosell/trailscraper.git"
```
Comment (nandahkrishna) [inline Formula/trailscraper.rb:10]: Please add this back.

File: Formula/trailscraper.rb [hunk 2]
```diff
@@ -17,31 +15,26 @@ class Trailscraper < Formula
     sha256 cellar: :any_skip_relocation, mojave:        "4ee380a299a7c96115c1cf9f89812d16243853fc2f02e9f84c2ab69fd6e89b90"
   end
 
-  depends_on "python@3.9"
+  depends_on "python@3.8"
```
Comment (nandahkrishna) [inline Formula/trailscraper.rb:18]: Why downgrade to Python 3.8?

File: Formula/trailscraper.rb [hunk 3]
```diff
@@ -3,11 +3,9 @@ class Trailscraper < Formula
 
   desc "Tool to get valuable information out of AWS CloudTrail"
   homepage "https://github.com/flosell/trailscraper"
-  url "https://files.pythonhosted.org/packages/ee/5f/67657d646b3a6fdd9059f72cf9f13cb9ed677d92c881bc83b67179c20178/trailscraper-0.6.4.tar.gz"
-  sha256 "3e076fc97addddfd8b64ea51a6cc9e2af9d9bffcc68423772ff6cb7b63a009d6"
+  url "https://github.com/flosell/trailscraper/archive/0.7.0.tar.gz"
+  sha256 "39fc2895629242dc72e0f3d3665b7fe53cc1596228578f5f7384cc6ea09d353e"
```
Comment (nandahkrishna) [inline Formula/trailscraper.rb:7]: Could you please change this back to the PyPI URL?

**Comment by nandahkrishna** [comment]
@flosell I'm not too familiar with `homebrew-pypi-poet` but you could simply use `brew bump-formula-pr` – it'll update the resources for you if the stable URL is from PyPI.

**Comment by flosell** [comment]
hi @nandahkrishna, thanks for the review! looks like the automation with poet got a bit creative there :) changed things back to how you suggested, will have a look if I can just use `brew bump-formula-pr` in the future...

**Comment by flosell** [comment]
Ok, fixed my automation and created a new one. Closing this one as a duplicate of #79185

**Pull request closed without merging**